examples/tfengine/generated/iam_members/iam_members/variables.tf

```diff
@@ -47,3 +47,12 @@ variable "organization_iam_members" {
     bindings     = map(list(string))
   }))
 }
+
+variable "service_account_iam_members" {
+  description = "IAM members for service accounts. Assigns additional non-authoritative IAM bindings to a list of service accounts."
```

Comment:
"Assigns additional non-authoritative IAM bindings to a list of service accounts." is a bit ambiguous as people might think it's granting any roles to a list of service accounts.

Consider rephrasing it to something like

"Assigns additional non-authoritative service account roles (e.g. roles/iam.serviceAccountUser) to a list of service accounts."

Reply:
for sure. fixed